kie-user-bom-parent/pom.xml

```diff
@@ -74,6 +88,33 @@
       </snapshots>
     </repository>
   </repositories>
```

Comment:
I've the added comment https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/407/commits/55a6c9e78e560c45465121f644dd36c4a107e467